packages/cli/helpers/build.ts

```diff
@@ -11,27 +11,6 @@ import { run } from '../../../helpers/compile/run'
 
 const copyFile = promisify(fs.copyFile)
 
-/**
- * The CLI wants to bundle everything, including @prisma/sdk and @prisma/studio.
- * For that reason, it sets all its dependencies as devDependencies.
- *
- * Problem: @prisma/studio has a peerDependency to @prisma/sdk. Because the CLI
- * doesn't have @prisma/sdk as dependency, but rather a devDependency, esbuild
- * will bundle @prisma/studio but mark its imports to @prisma/sdk as external.
- *
- * Solution: Since that is not our intent, and we want @prisma/studio to share
- * the same @prisma/sdk version that is bundled in the CLI, we have this plugin.
- */
-const resolveHelperPlugin: esbuild.Plugin = {
-  name: 'resolveHelperPlugin',
-  setup(build) {
-    // for any import of @prisma/sdk, resolve to this one
-    build.onResolve({ filter: /^@prisma\/sdk$/ }, () => {
-      return { path: require.resolve('@prisma/sdk') }
-    })
-  },
-}
-
```

Comment:
@millsp How is this not needed anymore?

Reply:
This is now done automatically via TS `paths`. If you look at the root `tsconfig.build.json`, I added paths for the build step, that info is used by `esbuild` at compile time, so it already knows where to find the `@prisma/sdk`.